examples/writing_sparse_global_test.go

```diff
@@ -99,24 +100,12 @@ func writeSparseGlobalArray() {
 	err = query.SetLayout(tiledb.TILEDB_GLOBAL_ORDER)
 	checkError(err)
 
-	// First submission
-	coords1 := []int32{1, 1, 2, 4}
-	data1 := []int32{1, 2}
-	_, err = query.SetBuffer("a", data1)
-	checkError(err)
-	_, err = query.SetCoordinates(coords1)
-	checkError(err)
-
-	// Perform the write.
-	err = query.Submit()
-	checkError(err)
-
-	// Submit second query
-	coords2 := []int32{2, 3}
```

Comment:
I think we can keep this but just create a second query object?

Reply:
Ok, will do, especially because in the case of `ExampleWritingSparseMultiple` the purpose of the test is cancelled

Reply:
Done